### ISSUE TITLE
Hidden sharkplanner UI blocking keyboard

### ISSUE THREAD

**Issue by AnY-cz** [okopanja/SharkPlanner#133]
When using sharkplanner and clicking on the gui with the mouse on the hide button, I get a key lock. Only the function keys remain
for sharkplanner. The key blocking is then for all the modules I own M2000 K50 FC3 Mig21. Then I have to exit the game to go back to the menu and there the keys work again. If I use the keyboard shortcut ctrl+shift+space to hide sharkplanner the error does not occur. I can't fix the error using ctrl+shift+space.

Translated with DeepL.com (free version)

https://github.com/okopanja/SharkPlanner/assets/31078339/92babb97-02c8-4e90-adbd-c0d3210b02cc

[dcs.log](https://github.com/okopanja/SharkPlanner/files/14896581/dcs.log)
[SharkPlanner.log](https://github.com/okopanja/SharkPlanner/files/14896582/SharkPlanner.log)

**Comment by okopanja** [comment]
Hi, I just saw your bug report. 
I will try to replicate and confirm.

**Comment by okopanja** [comment]
I checked briefly the logs: there is nothing abnormal that I see there.

About keyboard locking. there are two possible causes that I need to inspect:
- registered hotkeys that enable faster input when using both mouse and keyboard
- In order to support entry of the altitudes, I had to ensure that keyboard input into the module itself is blocked at the time when the SharkPlanner is displayed in order to avoid the keys to trigger both module and UI of shark planner. I did best to record the original state and restore it after the SharkPlanner gets hidden, but perhaps I have made a mistake. 

Meanwhile I will release the v1.6.1, but likely it also has the same issue.
